code/game/machinery/Sleeper.dm

```diff
@@ -201,13 +255,18 @@
 
 /obj/machinery/sleeper/process()
 	if (stat & (NOPOWER|BROKEN))
+		if(occupant)
+			occupant.in_stasis = null
+		stasis = FALSE
+		filtering = FALSE
+		stop_processing() //Shut down; stasis off, filtering off, stop processing.
 		return
 
-	if(filtering > 0)
+	if(filtering)
 		if(beaker)
 			if(beaker.reagents.total_volume < beaker.reagents.maximum_volume)
-				for(var/datum/reagent/x in occupant.reagents.reagent_list)
-					occupant.reagents.trans_to(beaker, 3)
+				for(var/datum/reagent/x in src.occupant.reagents.reagent_list)
+					src.occupant.reagents.trans_to(beaker, 10)
```

Comment:
Is this intended or the byproduct of not rebasing?

Reply:
Intended.